src/d3a/models/powerflow/pandapower.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import pandapower as pp
+from pandapower.plotting import to_html
+from d3a.models.powerflow import PowerFlowBase
+
+
+class PandaPowerFlow(PowerFlowBase):
+    def __init__(self, root_area, root_voltage=400):
+        self.network = pp.create_empty_network()
+        super().__init__(root_area, root_voltage)
+
+    def create_bus(self, area, voltage):
+        return pp.create_bus(self.network, vn_kv=voltage, name=area.name)
+
+    def add_external_grid(self, area):
+        return pp.create_ext_grid(self.network, bus=area.parent.bus, vm_pu=1.0, name=area.name)
+
+    def add_load_device(self, area, avg_power_w):
+        return pp.create_load(self.network, bus=area.parent.bus, p_mw=avg_power_w * 1e-6,
```

Comment:
It would be nice to have 1e-6 and 1e-3 as constants or small functions (namely convert_w_to_mw and convert_kw_to_mw respectively). That way we will be able to get rid of the hardcoded conversions, and to reuse them in other places too.